blaze-server/src/main/scala/org/http4s/server/blaze/Http1ServerStage.scala

```diff
@@ -118,36 +118,37 @@ class Http1ServerStage(service: HttpService,
     val rr = new StringWriter(512)
     rr << req.httpVersion << ' ' << resp.status.code << ' ' << resp.status.reason << "\r\n"
 
-    val respTransferCoding = Http1Stage.encodeHeaders(resp.headers, rr, true) // kind of tricky method returns Option[Transfer-Encoding]
+    Http1Stage.encodeHeaders(resp.headers, rr, true)
+
+    val respTransferCoding = `Transfer-Encoding`.from(resp.headers)
+    val lengthHeader = `Content-Length`.from(resp.headers)
     val respConn = Connection.from(resp.headers)
```

Comment:
here are the other traversals. I moved the other two to the same place so there are three altogether.